<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolPanel</class>
 <widget class="QWidget" name="ToolPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="Frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>129</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Tool_crayon">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_crayon.png</normaloff>Resources/Tools/tool_crayon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_gomme">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_gomme.png</normaloff>Resources/Tools/tool_gomme.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_rect_select">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>28</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_rectselect.png</normaloff>Resources/Tools/tool_rectselect.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_aero">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_aero.png</normaloff>Resources/Tools/tool_aero.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_pot">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_pot.png</normaloff>Resources/Tools/tool_pot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_none1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_baguette">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>28</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_baguette.png</normaloff>Resources/Tools/tool_baguette.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_selec_color">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>28</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_seleccolor.png</normaloff>Resources/Tools/tool_seleccolor.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_snap">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>28</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_snap.png</normaloff>Resources/Tools/tool_snap.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_none2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>28</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_move">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>52</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_move.png</normaloff>Resources/Tools/tool_move.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_grand">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>52</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_grand.png</normaloff>Resources/Tools/tool_grand.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_rotation">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>52</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_rotation.png</normaloff>Resources/Tools/tool_rotation.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_flip">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>52</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Tools/tool_flip.png</normaloff>Resources/Tools/tool_flip.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Tool_none3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>52</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
